build_tools/travis/install.sh

```diff
@@ -60,4 +60,7 @@ if [[ "$SKIP_TESTS" != "true" ]]; then
 
     # PyTorch
     conda install --yes pytorch torchvision -c pytorch
+
+    # Installation
+    python setup.py install
```

Comment:
I think you could add "if [[ "$SKIP_TESTS" != "true" ]] here.

Reply:
But @zhangguanheng66 the code is inside the ```if``` statement. Is this nessesary?